demo-pages.js

```diff
@@ -60,30 +79,5 @@ $_documentContainer.innerHTML = `<custom-style>
     }
```

Comment:
L77: `body &gt; div.layout.horizontal.center-justified {`

Should the `&gt;` be in there?